README.md

```diff
@@ -33,6 +27,8 @@ Use the Deploy Button below, you'll deploy the example using [Vercel](https://ve
 
 [![Deploy with Vercel](https://vercel.com/button)][vercel-deploy]
 
+## Step 2. Set up the project locally
+
 [Clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) that Vercel created for you and from the root directory of your local checkout.
```

Comment:
A typo or something wrong with grammar here. Not sure what was the intended meaning of this sentence.

Reply:
The meaning of this sentence is: 
Once you clone it, you'll have a lot of folders. From the root directory (not going into any of those folders), then you'll need to link your cloned code to Vercel: (commands that come after)

Re reading I can see how confusing that might seem, does my explanation sound simpler? 🤔

Reply:
I've updated it :) I think it sounds better now 👍